Fixes #14522: Added g support for Vv mode

deepakchethan opened the pull request:
* Moved the existing 'g' (r_core_seek) into 'G'

radare commented:
Why

> On 17 Aug 2019, at 04:54, dodococo <notifications@github.com> wrote:
> 
> Moved the existing 'g' (r_core_seek) into 'G'
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/radare/radare2/pull/14823
> 
> Commit Summary
> 
> Fixes #14522: Added g support for Vv mode
> File Changes
> 
> M libr/cons/dietline.c (26)
> M libr/core/vmenus.c (5)
> Patch Links:
> 
> https://github.com/radare/radare2/pull/14823.patch
> https://github.com/radare/radare2/pull/14823.diff
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.

deepakchethan commented:
> Why
> […](#)
> On 17 Aug 2019, at 04:54, dodococo ***@***.***> wrote: Moved the existing 'g' (r_core_seek) into 'G' You can view, comment on, or merge this pull request online at: #14823 Commit Summary Fixes #14522: Added g support for Vv mode File Changes M libr/cons/dietline.c (26) M libr/core/vmenus.c (5) Patch Links: https://github.com/radare/radare2/pull/14823.patch https://github.com/radare/radare2/pull/14823.diff — You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub, or mute the thread.

g in Vv was already assigned to it, so I moved it under G and used g for offset thing. Is that okay?

radare commented on libr/cons/dietline.c:
```diff
@@ -1341,9 +1341,14 @@ static inline void delete_till_end () {
 }
 
 static void __print_prompt () {
```
Missing space before (

deepakchethan replied:
Added

radare commented on libr/cons/dietline.c:
```diff
@@ -2391,26 +2396,7 @@ R_API const char *r_line_readline_cb(RLineReadCallback cb, void *user) {
-				if (len > 0 && (i + len) <= I.buffer.length) {
-					fwrite (I.buffer.data + i, 1, len, stdout);
-				}
+			        __print_prompt();
```
Missing space

deepakchethan replied:
Added
